package.json

```diff
@@ -33,6 +33,8 @@
     "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "lint:md": "remark  -f '*.md' 'doc/*.md'",
+    "lint:text": "textlint '*.md' 'doc/*.md'",
```

Comment:
Should be mentioned in the document about npm tasks (development.md?), isn't it?

Reply:
development.md updated. Fix c45ba05